sass/design_system.scss

```
@@ -23,112 +23,125 @@ body {
   min-height: 100%;
 }
 
-header,
-footer {
+.usa-header,
+.usa-footer {
   padding: 1rem 5rem;
   margin: 0;
-  background-color: $brand-darker;
+  background-color: color("primary-darker");
 }
 
-h1 {
+h1,
+.usa-prose > h1 {
   @include u-font("serif", 12);
 }
-h2 {
+h2,
+.usa-prose > h2 {
   @include u-font("sans", 11);
 }
-h3 {
+h3,
+.usa-prose h3 {
```

```suggestion
.usa-prose > h3 {
```
guessing we missed a descendant selector here